packages/vuefire/src/index.js

```diff
@@ -41,11 +41,13 @@ function bind ({ vm, key, ref, ops }, options) {
 
 export function firestorePlugin (
   Vue,
-  { bindName = '$bind', unbindName = '$unbind' } = {}
+  { bindName = '$bind', unbindName = '$unbind', createSnapshot } = {}
```

Comment:
I'm not sure about the naming because a snapshot has a particular meaning in Firestore, so let's try to find other alternatives. Maybe `serializeSnapshot`?

Reply:
maybe something with `populate` would be nice, since it's gonna get the firestore data and populate the vm.

Reply:
I missed this message, sorry :(
I think we should emphasize on the fact that it transforms the data coming from Firebase before we set it on the instance, that's why I think the verb should be related to that with something like serialize or serializer